pom.xml

```diff
@@ -378,6 +397,9 @@
           <skipTests>${skip.surefire.tests}</skipTests>
           <runOrder>balanced</runOrder>
           <reuseForks>false</reuseForks>
+          <systemPropertyVariables>
+            <jenkins.test.timeout>600</jenkins.test.timeout>
+          </systemPropertyVariables>
```

Comment:
Adding the timeout in both `failsafe` and `surefire` ensures that running test with `mvn verify` or `mvn test` makes the same timeout applying.

there was a timeout configuration earlier via argument passing, but it got revered to 0